docs/source/installation.rst

```diff
@@ -0,0 +1,110 @@
+.. _installation:
+
+Installation instructions
+=========================
+
+Note: T3 was only tested on Linux (Ubuntu_ 18.04.1 LTS) and Mac machines. We don't expect it to work on Windows.
+T3 can be installed on a server, as well as on your local desktop / laptop, submitting jobs to the server/s.
+Note that T3 requires installation of both `RMG <http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/index.html>`_, Reaction Mechanism Generator, and `ARC <https://reactionmechanismgenerator.github.io/ARC/installation.html>`_, Automated Rate calculator.
+
+
+.. _path:
+
+Clone and setup path
+^^^^^^^^^^^^^^^^^^^^
+
+- Download and install the `Anaconda Python Platform`__ for Python 3.7 or higher if you haven't already.
+- Get git if you don't have it already by typing sudo apt-get install git in a terminal.
+- Clone T3's repository to by typing the following command in the desired folder (e.g., under `~/home/Code/`)::
+
+    git clone https://github.com/ReactionMechanismGenerator/T3.git
+			  
+- Add T3 to your local path in .bashrc (make sure to change `~/Path/to/T3/` accordingly)::
+
+    export PYTHONPATH=$PYTHONPATH:~/Path/to/T3/
+
+__ anaconda_
+
+
+Install dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+- Install the latest DEVELOPER version of RMG (which has Arkane).
+  It is recommended to follow RMG's `Developer installation by source using Anaconda
+  <http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/index.html
+  #for-developers-installation-by-source-using-anaconda-environment>`_ instructions.
+  Make sure to add RMG-Py to your PATH and PYTHONPATH variables as explained in RMG's documentation.
+- Install the latest version of ARC by following `ARC's installation instructions 
+  <https://reactionmechanismgenerator.github.io/ARC/installation.html>`_. 
+  Make sure to add ARC to your PATH and PYTHONPATH variables as well as define servers as 
+  explained in ARC's documentation. 
+- If you'd like to use `AutoTST <https://github.com/ReactionMechanismGenerator/AutoTST>`_ in ARC (optional),
+  clone it in a separate folder and add it to your PYTHONPATH just as well.
+- Create the Anaconda environment for T3::
+
+    conda env create -f environment.yml
+
+  Activate the T3 environment every time before you run T3::
+
+     source activate t3_env
+
+
+Add T3 aliases to your .bashrc (for convenience)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some optional, but convenient, aliases are listed below (make sure to change `~/Path/to/T3/` accordingly).
```

Comment:
here we have `~/`, below we don't
although it's a dummy, let's be consistent

Reply:
I made them all include `~/` now. Good catch thanks!